hack/testdata/rollingupdate-statefulset-rv2.yaml

```diff
@@ -0,0 +1,43 @@
+apiVersion: v1
```

Comment:
We do not need to have the service definition be part of the fixture data, just the statefulset should suffice.